repoclosure/yum_el7.conf

```
@@ -45,32 +45,32 @@ baseurl=https://yum.puppetlabs.com/el/7/PC1/x86_64/
 name=el7-puppet-6
 baseurl=https://yum.puppetlabs.com/puppet6/el/7/x86_64/
 
-[el7-foreman-rails-nightly]
-name=Foreman Rails nightly EL7
-baseurl=http://koji.katello.org/releases/yum/tfm-ror52/el7/x86_64/
+[el7-foreman-rails-1.24]
+name=Foreman Rails 1.24 EL7
+baseurl=http://koji.katello.org/releases/yum/foreman-rails-1.24/el7/x86_64/
 
 # Used as lookaside repos for layered repos (plugins)
-[el7-foreman-nightly]
-name=Foreman nightly EL7
-baseurl=http://koji.katello.org/releases/yum/foreman-nightly/RHEL/7/x86_64/
+[el7-foreman-1.24]
+name=Foreman 1.24 EL7
+baseurl=http://koji.katello.org/releases/yum/foreman-1.24/RHEL/7/x86_64/
 
-[el7-foreman-plugins-nightly]
-name=Foreman Plugins nightly EL7
-baseurl=http://koji.katello.org/releases/yum/foreman-plugins-nightly/RHEL/7/x86_64/
+[el7-foreman-plugins-1.24]
+name=Foreman Plugins 1.24 EL7
+baseurl=http://koji.katello.org/releases/yum/foreman-plugins-1.24/RHEL/7/x86_64/
 
-[el7-foreman-client-nightly]
-name=Foreman Client nightly EL7
-baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/el7/x86_64/
+[el7-foreman-client-1.24]
+name=Foreman Client 1.24 EL7
+baseurl=http://koji.katello.org/releases/yum/foreman-client-1.24/el7/x86_64/
 
 # Katello repos for lookaside
-[el7-katello-nightly]
-name=Katello nightly EL7
-baseurl=http://koji.katello.org/releases/yum/katello-nightly/katello/el7/x86_64/
+[el7-katello-3.14]
+name=Katello 3.14 EL7
+baseurl=http://koji.katello.org/releases/yum/katello-3.14/katello/el7/x86_64/
 
-[el7-katello-pulp-nightly]
-name=Katello Pulp nightly EL7
-baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/stable/2.19/7Server/x86_64/
+[el7-katello-pulp-2.21]
```

I think we use 3.14 here because in Jenkins/obal we don't care about the exact version. We'll just say "we need the pulp config for katello 3.14, whatever that may be".
```suggestion
[el7-katello-pulp-3.14]
```